contributors/join-us/staff-recruiting/README.md

```diff
@@ -4,21 +4,60 @@ description: 1% complete.
+
+Your Recruiter will be based on the role and team. This may differ depending on Recruiting Team bandwidth, and will be communicated on a case-by-case basis. 
+* Technical Individual Contributor Roles -- Teresa Novoa 
+* All others -- Natalie Jew 
+
+## Recruiter In-Take Session 
+
+Your Recruiter will schedule an in-take session after you’ve completed your Job Description. During this call, your Recruiter will work with you to ensure alignment on the level, ideal candidate profile, the interview process and next steps. 
+
+### Role Leveling 
+
+You can view the current [v1 Job Levels here](https://docs.google.com/spreadsheets/d/1kNsl7JUWDQ8MhWonys9dR3osrxEWYkKVIoECfFKSQzw/edit#gid=0). Note: HR is also working with MLT to build v2 which will help provide additional clarity. If you are unsure about how to use or assign a level to a new job req, your recruiter can help you walk through this as part of the Recruiter In-Take. 
+
+### Ideal Candidate Profile (ICP) 
+
+The ICP should be a clear list of key skills, experiences and/or mindsets that the Hiring Manager believes represent the Ideal Candidate. The Hiring Manager will be asked to stack rank the ICP list and use ‘P0’ to indicate "must haves" and 'P1' to indicate "nice to have" the ideal skills, experiences and mindsets. 
+
+The ICP will be used to align the sourcing, interview panel focus areas and onboarding goals for this new role. For example, here’s an ICP for a [Lead Customer Journey Engineer](https://jobs.lever.co/mattermost/3c1dd4ff-b135-4bb7-8b14-73c4a8cd674f). The list below reflects the top skills and experience to source for candidates, the area of focus for the interview panel and what the eventual new hire's onboarding goals will be based on.  
 
-**Responsibilities:** 
+P0 - 2+ Years experience leading a team of engineers
+P1 - Experience as an Engineer or a Lead for a team doing front-end/UI-UX work in JavaScript, React, etc.
 
-* What will the candidates need to do be successful in their role in our culture \(e.g. hire and manage team building new product line\)? 
-* What will the interview panel be interviewing for in their meetings? 
+### Changing the ICP 
 
-**Requirements:** 
+The ICP is also a helpful indicator of talent pools. If a role is not gaining enough candidate pipeline or the candidate fit is off, the ICP enables Hiring Managers to efficiently determine appropriate tradeoffs as it relates to the P0 or P1 skills, experience and mindsets. For example, a Recruiter may find that we have filtered multiple candidates who do not have a specific P0 skill. As a result, a Hiring Manager may determine that it may be worth deprioritizing a specific qualification from P0 to P1 in order to widen the candidate pool. If an ICP is ever changed, the Recruiter will ensure that the hiring team is aligned to the change. 
```

Comment:
```suggestion
The ICP is also a helpful indicator of talent pools. If a role is not gaining enough candidate pipeline or the candidate fit is off, the ICP enables Hiring Managers to efficiently determine appropriate tradeoffs as it relates to the "P0" or "P1" skills, experience, and mindsets. 

For example, a Recruiter may find that we have filtered multiple candidates who do not have a specific "P0" skill. As a result, a Hiring Manager may determine that it may be worth deprioritizing a specific qualification from "P0" to "P1" in order to widen the candidate pool. If an ICP is ever changed, the Recruiter will ensure that the hiring team is aligned to the change.
```